src/pages/Facility/services/HealthcareServiceShow.tsx

```diff
@@ -26,7 +26,16 @@ function LocationCard({
   const { t } = useTranslation();
 
   return (
-    <Card className="transition-all duration-200 hover:border-primary/50 hover:shadow-sm rounded-md">
+    <Card
+      onClick={() =>
+        navigate(
+          service_type === InternalType.pharmacy
```

Comment:
render service_requests/ link only for lab 👍 Otherwise this could become an issue if we add a new service_type.